translations/management/commands/bulk_translate.py

```diff
@@ -2,7 +2,7 @@
 from django.conf import settings
 from django.db.models import Q, OuterRef, Exists
 from translations.models import Translation
-from integrations.services.google_translate.integration import Translate
+from integrations.clients.google_translate import Translate
```

Comment:
Everything looks good with this change but just want to confirm that we've run the `bulk_translate` management command locally.